pages/m/[measurement_uid].js

```diff
@@ -0,0 +1,259 @@
+import axios from 'axios'
+import Head from 'next/head'
+import { Container, theme } from 'ooni-components'
+import PropTypes from 'prop-types'
+import React, { useMemo, useState } from 'react'
+import useSWR from 'swr'
+import { getLocalisedRegionName } from '/utils/i18nCountries'
+
+import CommonDetails from 'components/measurement/CommonDetails'
+import CommonSummary from 'components/measurement/CommonSummary'
+import DetailsHeader from 'components/measurement/DetailsHeader'
+import FeedbackBox from 'components/measurement/FeedbackBox'
+import HeadMetadata from 'components/measurement/HeadMetadata'
+import Hero from 'components/measurement/Hero'
+import MeasurementContainer from 'components/measurement/MeasurementContainer'
+import MeasurementNotFound from 'components/measurement/MeasurementNotFound'
+import SummaryText from 'components/measurement/SummaryText'
+
+import NavBar from 'components/NavBar'
+import useUser from 'hooks/useUser'
+import ErrorPage from 'pages/_error'
+import { useIntl } from 'react-intl'
+import { fetcher } from '/lib/api'
+
+const pageColors = {
+  default: theme.colors.base,
+  anomaly: theme.colors.yellow9,
+  reachable: theme.colors.green8,
+  error: theme.colors.gray6,
+  down: theme.colors.gray6,
+  confirmed: theme.colors.red7
+}
+
+export async function getServerSideProps({ query }) {
+  let initialProps = {
+    error: null,
+    userFeedback: null
+  }
+
+  const measurement_uid = query?.measurement_uid
+  // If there is no measurement_uid to use, fail early with MeasurementNotFound
+  if (typeof measurement_uid !== 'string' || !measurement_uid.match(/[a-zA-Z0-9._-]{40,100}/)) {
```

Comment:
I don't think a valid `measurement_uid` can contain the `-` character, but @FedericoCeratto can confirm.

I'm also not sure we want to be doing this type of validation of the format of the measurement_uid inside of explorer.